doc/developer/design/00000000_template.md

```diff
@@ -3,70 +3,99 @@
 - Associated: (Insert list of associated epics, issues, or PRs)
 
 <!--
+The goal of a design document is to thoroughly discover problems and
+examine potential solutions before moving into the delivery phase of
+a project. In order to be ready to share, a design document must address
+the questions in each of the following sections. Any additional content
+is at the discretion of the author.
+
 Note: Feel free to add or remove sections as needed. However, most design
 docs should at least keep the suggested sections.
 -->
 
-## Context
+## The Problem
 
 <!--
-Bring the reader up to speed enough, such that they can understand the
-following goals and descriptions.
-
-An important reason for this is helping future readers understand the
-assumptions that went into the design, and in turn the goals and design itself.
+What is the user problem we want to solve?
 
-Be sure to capture the customer impact/customer problem, which should be the
-motivation for the proposed design!
+The answer to this question should link to at least one open Github
+issue describing the problem.
 -->
 
-## Goals
+## Success Criteria
 
 <!--
-Enumerate the concrete goals that are in scope for the project.
+What does a solution to this problem need to accomplish in order to
+be successful?
+
+The criteria should help us verify that a proposed solution would solve
+our problem without naming a specific solution. Instead, focus on the
+outcomes we hope results from this work. Feel free to list both qualitative
+and quantitative measurements.
```

Comment:
I personally like the idea of an Exit Criteria rather than a Success Criteria which is a subset. Once the implementation is underway, we might learn more which might change or limit what we hoped to achieve. Being cognizant about the risks and the potential end results is helpful I think.

Reply:
If the above works for you, I'd love to resolve this comment.

Reply:
Sounds good 👍